apps/platform/src/app/(main)/members/@invited/page.tsx

```diff
@@ -0,0 +1,228 @@
+'use client'
+
+import React, { useCallback } from 'react'
+import dayjs from 'dayjs'
+import { CloseCircleSVG } from '@public/svg/shared'
+import { useAtom, useAtomValue } from 'jotai'
+import type { GetMembersResponse } from '@keyshade/schema'
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow
+} from '@/components/ui/table'
+import AvatarComponent from '@/components/common/avatar'
+import { Button } from '@/components/ui/button'
+import {
+  cancelInviteOpenAtom,
+  resendInviteOpenAtom,
+  selectedMemberAtom,
+  selectedWorkspaceAtom
+} from '@/store'
+import { InfiniteScrollList } from '@/components/ui/infinite-scroll-list'
+import ControllerInstance from '@/lib/controller-instance'
+import {
+  Tooltip,
+  TooltipArrow,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger
+} from '@/components/ui/tooltip'
+import CancelInvitationDialog from '@/components/members/cancelInvitationDialog'
+import ResendInvitationDialog from '@/components/members/resendInvitationDialog'
+
+function MemberRow({
+  member,
+  onCancelInvitation,
+  onResendInvitation
+}: {
+  member: GetMembersResponse['items'][number]
+  onCancelInvitation: (member: GetMembersResponse['items'][number]) => void
+  onResendInvitation: (member: GetMembersResponse['items'][number]) => void
+}) {
+  return (
+    <TableRow className="group hover:bg-transparent" key={member.id}>
+      <TableCell className="w-[30%] text-left">
+        <div className="flex items-center">
+          <AvatarComponent
+            className="h-10 w-10"
+            name={member.user.name || ''}
+            profilePictureUrl={member.user.profilePictureUrl}
+          />
+          <div className="ml-3">
+            <div className="text-base font-medium text-white">
+              {member.user.name}
+            </div>
+            <div className="text-xs font-normal text-white">
+              {member.user.email}
+            </div>
+          </div>
+        </div>
+      </TableCell>
+      <TableCell className="w-[30%] text-left">
+        {dayjs(member.createdOn).format('MMM D, YYYY')}
+      </TableCell>
+      <TableCell className="w-[40%] text-left">
+        <div className="flex w-[8rem] items-center justify-center rounded-full border border-purple-200 bg-[#3B0764] px-4 py-2 text-purple-200">
+          {member.roles[0].role.name}
+        </div>
```

Comment:
Extract this into a separate file. I think you can reuse this component across both the tabs. Use a `tabType` prop in the item with values `pending` and `joined`